logstash-core/spec/logstash/shutdown_watcher_spec.rb

```diff
@@ -1,12 +1,12 @@
 # encoding: utf-8
 require "spec_helper"
-require "logstash/shutdown_controller"
+require "logstash/shutdown_watcher"
 
-describe LogStash::ShutdownController do
+describe LogStash::ShutdownWatcher do
 
   let(:check_every) { 0.01 }
   let(:check_threshold) { 100 }
-  subject { LogStash::ShutdownController.new(pipeline, check_every) }
+  subject { LogStash::ShutdownWatcher.new(pipeline, check_every) }
```

Comment:
I know, but I really don't like "described_class", it's a terrible name, while having the class explicitly there makes it more readable IMO.
That said, I can change you feel strongly on this.